templates/ReactReduxSpa/tsconfig.json

```diff
@@ -5,7 +5,12 @@
     "jsx": "preserve",
     "experimentalDecorators": true,
     "sourceMap": true,
-    "skipDefaultLibCheck": true
+    "skipDefaultLibCheck": true,
+    "lib": ["es6", "dom"],
+    "typeRoots": [ "node_modules/@types" ],
+    "types": [ "react", "react-dom", "react-redux", "react-router", "react-router-redux",
+     "redux", "redux-thunk", "source-map", "uglify-js", "webpack", "webpack/webpack-env",
```

Comment:
Even though it is not yet at npm: `webpack-env` has been outsourced into an additional package ([DefinitelyTyped #11684](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/11684)).